db/migrate/20240729192228_add_convert2rhel_to_host_facets.rb

```diff
@@ -0,0 +1,9 @@
+class AddConvert2rhelToHostFacets < ActiveRecord::Migration[6.1]
+  def up
+    add_column :katello_subscription_facets, :convert2rhel_through_foreman, :int4
```

Comment:
I didn't see this before, but isn't there a boolean column type that's technically more correct?